tools/form-recognizer/init.sh

```diff
@@ -2,5 +2,6 @@
 # Seed the shared folder that contains the ocr model on startup
 if [ ! -f /shared/init.lock ]; then
   touch /shared/init.lock
+  rm -rf /shared/{*,.*}
```

Comment:
Attempt to clear out old model from PVC. If this doesn't work when this container is deployed, we'll need to drop and recreate the PVC.